cmake/Modules/CMakeDetectOsXArchs.cmake

```diff
@@ -0,0 +1,35 @@
+# Detects all supported OSX architectures by reading libSystem.B.dylib
+# Sets the possible values in DETECTED_OSX_ARCHS
+# If a NO{ARCH} variable is set, then {ARCH} will be ignored
+
+MACRO(DetectOsXArchs)
+    # find lipo
+	FIND_PROGRAM(LIPO NAMES apple-lipo lipo)
```

Comment:
okay but standard name is okay - if we use the lipo from toolchain first (see comment below)